README.md

```diff
@@ -28,13 +28,22 @@ Verify the gem was installed and tasks are available:
 
     $ bundle exec cap -T opscomplete
 
-Now, add some [hooks](#using-capistrano-hooks) in your capistrano configuration. An example configuration could look like this:
+Now, add some [hooks](#using-capistrano-hooks) in your capistrano configuration (e.g. `deploy.rb`).
+An example configuration could look like this:
 
 ```ruby
 # After unpacking your release, before bundling, compiling assets, ...
 after 'deploy:updating', 'opscomplete:ruby:ensure'
 ```
 
+and in case you enabled [`Procfile support`](https://makandracards.com/opscomplete/67829-procfile-support) this can be enabled:
+
+```ruby
+# Update and Restart supervisor config
+after 'deploy:updating', 'opscomplete:supervisor:gen_config'
```

Comment:
maybe use `deploy:updated` here, so it runs after bundler

Reply:
shouldn't matter much if this runs before or after `bundle install` since `supervisor_gen_config` is just a binary we call and not running in our bundle anyways (internally this is implemented in ruby too, but in a separate gem bundle and ruby version).